src/main/webapp/app/modeling-submission/modeling-submission.component.ts

```diff
@@ -126,7 +130,13 @@ export class ModelingSubmissionComponent implements OnInit, OnDestroy, Component
                                 this.prepareAssessmentData();
                             });
                             this.complaintService.findByResultId(this.result.id).subscribe(res => {
-                                this.hasComplaint = !!res.body;
+                                if (res.body) {
```

Comment:
Works, but this is not very rxjs like.
It would be better to do a `pipe(filter(res => !!res.body))` before the subscribe.

Reply:
Good point. Done

Reply:
It's still the same. Maybe you forgot to push?